frontend/src/components/assessments/assessment-creation/CreateAssessementButton.tsx

```diff
@@ -1,33 +1,25 @@
 import React from "react";
-import { useHistory } from "react-router-dom";
 import { Button } from "@chakra-ui/react";
 
 import { PlusOutlineIcon } from "../../../assets/icons";
 import { CREATE_ASSESSMENT } from "../../../constants/Routes";
+import RouterLink from "../../common/RouterLink";
 
 const CreateAssessementButton = (): React.ReactElement => {
-  const history = useHistory();
   const date = new Date();
   const options: Intl.DateTimeFormatOptions = {
     month: "long",
     day: "numeric",
     year: "numeric",
   };
   const formattedDate = date.toLocaleDateString("en-US", options);
-  const navigateToCreateAssessmentPage = () =>
-    history.push(CREATE_ASSESSMENT, { date: formattedDate });
 
   return (
-    <>
-      <Button
-        my={2}
-        onClick={navigateToCreateAssessmentPage}
-        rightIcon={<PlusOutlineIcon />}
-        variant="primary"
-      >
+    <Button my={2} rightIcon={<PlusOutlineIcon />} variant="primary">
+      <RouterLink to={`${CREATE_ASSESSMENT}/${formattedDate}`}>
```

Comment:
is there a reason we use the formatted date here?

design-wise it looks a little funky to me

also idk if its an issue if ppl can tamper the url (e.g. manually set to `March%2026,%202022`) - maybe there's a better way to hash the value? just some thoughts
<img width="166" alt="image" src="https://user-images.githubusercontent.com/69697180/227793288-7842061e-1f3f-461e-a028-eca682495364.png">

Reply:
you know what i think im dumb - i don't think we need the date in params at all, i think we can just call it inside the `CreateAssessmentHeader`